### ISSUE TITLE
This plugin does not seem to send any logs

### ISSUE THREAD

**Issue by garrypolley** [SpectrumApp/spectrum-chrome#1]
I'm using the chrome plugin with chrome: `Version 49.0.2623.112 (64-bit)`

I have it configured like so: 

<img width="403" alt="screen shot 2016-04-20 at 10 06 11 am" src="https://cloud.githubusercontent.com/assets/992175/14686788/f696afa2-06ff-11e6-846e-575011ecacd3.png">

I then have the application configured like so: 

<img width="429" alt="screen shot 2016-04-20 at 1 58 53 pm" src="https://cloud.githubusercontent.com/assets/992175/14686804/07ade12a-0700-11e6-8430-74596dfed8e1.png">

Changing the IP to be localhost does not change me not getting logs into spectrum. 

Any ideas on what I'm doing wrong?

**Comment by frankwiles** [comment]
They should be showing up in Level 'DEBUG' and SubLevel 'browser' with any code that calls `console.log` or `console.dir`.  

You also have to "enable" the extension by clicking it in the browser bar, it'll go from a grayscale icon to a fully colored Spectrum logo.

**Comment by garrypolley** [comment]
Thanks for the fast response. I do have the extension enabled.  The application I'm trying to use it for the most is an Angular based application. 

I'm not seeing anything running on port 9099 when I run an `lsof` or `netstat` check. I currently use a mac.

**Comment by frankwiles** [comment]
Hmmm, I don't know that we've specifically tested with Angular before.  We tend to use ReactJS.  So let's do some divide and conquer here and see if the problem is in the REST endpoint or the Chrome extension.  One easy way to do that is just curl, from [this example here](http://www.devspectrum.com/help/spectrum-rest-api/).  So try this:

```
curl -X POST -H "Content-Type: application/json" -d '{"id": "863ef2c3ea134841b87072", "level": "DEBUG", "message": "The daemon process has generated an error code 555", "number": 253, "sublevel": "browser", "timestamp": "2016-02-22 11:12:39.873108" }' http://localhost:9099/
```

If that works we know it's an issue with the extension and Angular.  Do you know if Angular perhaps hijacks console itself or maybe another Angular specific debugging extension?

**Comment by garrypolley** [comment]
Thanks for the help.  It looks like a combination of things. 
1. I needed to restart the specturm app after I had reconfigured my JavaScript Logs rest stream
2. Our application is not using console.info/debug etc when it logs (Angular related)
3. I don't think my local machine was allowing Chrome to send a request to it when it was anything other than `0.0.0.0` and the correct port.  

Thanks again for the help, I think that fixes all the problems I had.  The curl command really helped because it let me know that I had Spectrum set-up correctly.
